introduction.md

```diff
@@ -1 +1,5 @@
 Hello,  my name is Krzysztof. My favorite Code School path is Electives.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?